pkg-scripts/postinstall-rpm.sh

```diff
@@ -7,6 +7,8 @@ CONFIG_PATH=/etc/cagent/cagent.conf
 versionsCount=$1
 
 if [ ${versionsCount} == 1 ]; then # fresh install
+    chown cagent:cagent -R /var/lib/cagent
+    chmod 6777 /var/lib/cagent/jobmon
```

Comment:
what is the reason to have both `setgid` and `setuid`? jobmon can be run on a different user and push files with <666 permission?

Reply:
Yes, any user should be able to run jobmon, which will save the new file into that folder. Later cagent binary will read and remove it from folder. Also, there is one edge-case when jobmon can remove file created by other jobmon run, that exited unexpectedly.